docs/testrunner-toolkit/installation.md

```diff
@@ -67,6 +67,11 @@ There are multiple ways to install the Sauce Labs Testrunner Toolkit (colloquial
 * ```bash title="Using NPM"
   npm install -g saucectl
   ```
+  
+* ```bash title="Using NPM and SAUCECTL_INSTALL_BINARY"
+  SAUCECTL_INSTALL_BINARY=https://company.domain.com/saucectl_0.32.2_mac_64-bit.tar.gz npm install -g saucectl
+  ```
+Using the `SAUCECTL_INSTALL_BINARY` environment variable, you can control how `saucectl` is installed. You can use it in case you need to download `saucectl` from a known source or in case you use `npx saucectl`.
```

Comment:
Hey @valentyn88 -- 1) What is `npx saucectl`? (I am just curious) 2) How would users know whether they "need to download from a known source"? Perhaps we can be a little more prescriptive with this note? For example, do some users have corporate restrictions that prohibit downloads using cURL or NPM because the source could be unknown and therefor less secure and does this binary address that risk? If so, should this be the first option? 
Thanks for clarification -- I am still getting up to speed! :)

Reply:
1) `npx` is a tool that use to execute packages. `npx saucectl ` means that we run `saucectl` without installation. `npm install -g saucectl` means that we install `saucectl` globally. 2) Security-conscious customers would like to download `saucectl` from a known source. What our customers can do is host an archive with our binary on their servers and install `saucectl` from these trusted domains. Usually, they know about the firewall that they have.

Reply:
Thanks!